src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -3,64 +3,107 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { WorkPackage } from 'shared';
+import { WorkPackage, wbsPipe } from 'shared';
 import { percentPipe, fullNamePipe, datePipe, weeksPipe } from '../../../utils/pipes';
 import WbsStatus from '../../../components/WbsStatus';
-import PageBlock from '../../../layouts/PageBlock';
-import { Box, Grid } from '@mui/material';
+import { Box, Divider, Grid, Link, Stack, Typography } from '@mui/material';
 import DetailDisplay from '../../../components/DetailDisplay';
 import WorkPackageStageChip from '../../../components/WorkPackageStageChip';
 import { timelinePipe } from '../../../utils/pipes';
+import { Construction, Work } from '@mui/icons-material';
+import ScheduleIcon from '@mui/icons-material/Schedule';
+import StackedLineChartIcon from '@mui/icons-material/StackedLineChart';
+import DoneAllIcon from '@mui/icons-material/DoneAll';
+import { Link as RouterLink } from 'react-router-dom';
+import { routes } from '../../../utils/routes';
 
 interface WorkPackageDetailsProps {
   workPackage: WorkPackage;
+  dependencies: WorkPackage[];
 }
 
-const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage }) => {
+const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage, dependencies }) => {
   return (
-    <PageBlock
-      title={'Work Package Details'}
-      headerRight={
-        <Box>
-          {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
-          <WbsStatus status={workPackage.status} />
-        </Box>
-      }
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Lead" content={fullNamePipe(workPackage.projectLead)} paddingRight={2} />
+    <>
+      <Grid container columnSpacing={2} justifyContent="flex-start" sx={{ mb: 2, mt: 1 }}>
+        <Grid item display="flex" alignItems="center">
+          <Typography variant="h5">Work Package Details</Typography>
         </Grid>
+        <Grid item display="flex" alignItems="center">
+          <Box>{workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}</Box>
+        </Grid>
+
+        <Grid item display="flex" alignItems="center">
+          <Box>
+            <WbsStatus status={workPackage.status} />
+          </Box>
+        </Grid>
+      </Grid>
 
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Start Date" content={datePipe(workPackage.startDate)} paddingRight={2} />
+      <Grid container spacing={2}>
+        <Grid item display="flex" alignItems="center" xs={12} sm={6} md={3}>
+          <Construction sx={{ mr: 2 }} />
```

Comment:
why dont we get rid of the detail display and just use a transparent chip with this

Reply:
wdym, how would that look? This is just mimicing what we do in project details rn

Reply:
this ticket is just getting everything to look standard, and this code is literally copied from what anthony did in Project details. If you wanna redesign it again, we should do that in another epic, so we can change everything and roll out the change all at once.